Lib/AspNet/Microsoft.WindowsAzure.Storage/project.json

```diff
@@ -1,91 +1,74 @@
 ﻿{
-    "version": "7.0.1.0",
+    "version": "7.1.1.0",
     "authors": [ "Microsoft Corporation" ],
     "description": "Azure Storage SDK for CoreCLR Preview",
     "dependencies": {
         "Microsoft.Data.OData": "5.6.4",
         "Microsoft.Data.Services.Client": "5.6.4",
         "System.Spatial": "5.6.4",
-        "Newtonsoft.Json":  "6.0.8"
+        "Newtonsoft.Json": "6.0.8"
```

Comment:
btw - `Newtonsoft.Json 9.0.1` is latest and compat with `netstandard1.6/netcoreapp1.0`